src/views/EditSidebar.vue

```diff
@@ -107,7 +107,7 @@
 				@update-end-timezone="updateEndTimezone"
 				@toggle-all-day="toggleAllDay" />
 
-			<InvitationResponseButtons v-if="isViewedByAttendee && userAsAttendee && !isReadOnly"
+			<InvitationResponseButtons v-if="isViewedByAttendee"
```

Comment:
Why remove the `userAsAttendee` check from here? Is this an oversight or on purpose?

Reply:
`isViewedByAttendee` already checks `userAsAttendee`